CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 0.16.1
```

Comment:
Why are you incrementing both the minor version and the patch version here? I'm wondering if this should be `0.16.0` ?